python/spacetime/store_server.py

```diff
@@ -90,6 +100,8 @@ def get(self, sim):
         ret["new"] = all_new
         ret["updated"] = all_updated
         ret["deleted"] = all_deleted
+
+        reset_timer(sim)
```

Comment:
Should probably put it at the start of the function. Dont want gc to clean up while an update is happening.

Same for all reset_timer calls.